scripts/run_typedoc.sh

```diff
@@ -18,6 +18,9 @@ TOOL_APIDOCGEN="go run ./tools/tscdocgen/*.go"
 PULUMI_DOC_TMP=`mktemp -d`
 PULUMI_DOC_BASE=./reference/pkg/nodejs/@pulumi
 
+# Set this to 1 to run all generation in parallel.
+PARALLEL=1
```

Comment:
this is useful locally as it makes things about 3x faster on my machine.  On a build machine it sucks since all the output is interleaved.